### PR TITLE
Changes for wordpress Version Upgrade to 6.2.2

### PR DESCRIPTION
**Pull request by shivanisinghnitp**
Changes for wordpress Version Upgrade to 6.2.2

### CODE REVIEW

File: GenerateDockerFiles/wordpress/wordpress/wordpress_src/wordpress-azure/license.txt [hunk 1]
```diff
@@ -1,6 +1,6 @@
 WordPress - Web publishing software
```
Comment (ZubaeyrMSFT) [inline GenerateDockerFiles/wordpress/wordpress/wordpress_src/wordpress-azure/license.txt:1]: Did you update the w3tc plugin with changes required to make blob storage work in national clouds?

Comment (shivanisinghnitp) [reply GenerateDockerFiles/wordpress/wordpress/wordpress_src/wordpress-azure/license.txt:1]: yes there were 3 files with those changes, i referred this PR and did the changes - https://github.com/Azure-App-Service/ImageBuilder/pull/370/files

Comment (shivanisinghnitp) [reply GenerateDockerFiles/wordpress/wordpress/wordpress_src/wordpress-azure/license.txt:1]: Havent made the changes in Mariadb tho as u stated it is no longer used

Comment (ZubaeyrMSFT) [reply GenerateDockerFiles/wordpress/wordpress/wordpress_src/wordpress-azure/license.txt:1]: Can you also confirm if you have tested the following:
1. functioning of Redis cache 
2. CDN, AFD & Blob Storage
3. Upgrading of existing App with current prod image, with this new image.

Also, can you please share the test image details?

**Review by ZubaeyrMSFT — APPROVED** [review]

**Pull request merged**